examples/Superresolution/data_sampler.py

```diff
@@ -0,0 +1,123 @@
+import cv2
+import argparse
+import numpy as np
+import zipfile
+import random
+from tensorpack import RNGDataFlow, MapDataComponent, dftools
+
+"""
+DBDIR=/graphics/projects/scratch/datasets/mscoco/
+python data_sampler.py --lmdb ${DBDIR}val2017.lmdb --input ${DBDIR}val2017.zip --create
+python data_sampler.py --lmdb ${DBDIR}train2017.lmdb --input ${DBDIR}train2017.zip --create
+"""
```

Comment:
Instructions are in readme already. better to not duplicate them

Reply:
fixed